homepage.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>LessWrong Community Weekend</title>
+    <style>
+      /* Reset */
+      html {
+        box-sizing: border-box;
+        font-size: 16px;
+      }
+
+      *,
+      *:before,
+      *:after {
+        box-sizing: inherit;
+      }
+
+      body,
+      h1,
+      p,
+      ul {
+        margin: 0;
+        padding: 0;
+        font-weight: normal;
+      }
+
+      img {
+        max-width: 100%;
+        height: auto;
+      }
+
+      /* Style */
+      .col {
+        display: flex;
+        flex-direction: column;
+        gap: 8px;
+      }
+
+      body {
+        color: #212121;
+
+        font-family: Arial, Helvetica, sans-serif;
+
+        padding: 8px;
+
+        align-items: center;
+        justify-content: center;
+        height: 100dvh;
+      }
+
+      body > * {
+        max-width: 800px;
+      }
+
+      h1 {
+        text-align: center;
+        font-weight: 500;
+      }
+
+      ul {
+        padding-left: 16px;
+        list-style: disc;
+      }
+
+      img {
+        border-radius: 8px;
+        box-shadow: 2px 2px 8px gray;
+      }
+
+      a.button {
+        text-decoration: none;
+        color: white;
+        background-color: #5f9b65;
+        padding: 8px;
+        font-variant-caps: small-caps;
+        text-align: center;
+
+        border-radius: 8px;
+        box-shadow: 2px 2px 8px gray;
+      }
+    </style>
+  </head>
+  <body class="col" style="gap: 32px">
+    <h1>LessWrong Community Weekend 2025</h1>
+    <a
+      href="https://images.trvl-media.com/lodging/18000000/17810000/17804800/17804719/19212e43.jpg?impolicy=resizecrop&rw=1200&ra=fit"
+      target="_blank"
+      rel="noopener noreferrer"
+    >
+      <img
+        src="https://images.trvl-media.com/lodging/18000000/17810000/17804800/17804719/19212e43.jpg?impolicy=resizecrop&rw=1200&ra=fit"
+        alt="A picture of the hostel that hosts the LWCW"
+    /></a>
+    <div class="col">
+      <p>
+        Join the 12th Less Wrong Community Weekend (LWCW) in Berlin. This is the
+        world’s largest rationalist social gathering which brings together 250+
+        aspiring rationalists from across Europe and beyond for 4 days of
+        intellectual exploration, socialising and fun.
+      </p>
+      <p>
+        We will be taking over the whole hostel with a huge variety of spaces
+        inside and outside to talk, relax, dance, play, learn, teach, connect,
+        cuddle, practice, share ... - simply enjoy life together our way.
+      </p>
+    </div>
+    <div class="col">
+      <a href="https://test.lwcw.org" class="button">
```

Comment:
I think I would not post that link, don't see much upside (to be fair I also don't have any strong specific reasons not to, just a general feeling that I prefer not publishing something unfinished).

Reply:
Sure :+1: